collection/collector.go

```diff
@@ -5,14 +5,23 @@ import (
 	"time"
 
 	"github.com/odpf/raccoon/identification"
-	pb "github.com/odpf/raccoon/proto"
 )
 
+type Event struct {
+	Type       string
+	EventBytes []byte
+}
+
 type CollectRequest struct {
 	ConnectionIdentifier identification.Identifier
 	TimeConsumed         time.Time
 	TimePushed           time.Time
-	*pb.SendEventRequest
+	SentTime             time.Time
```

Comment:
https://github.com/odpf/raccoon/issues/35#issuecomment-1141682794
Inlining with this comment for reference